src/scripts/main.js

```diff
@@ -1,3 +1,98 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.field');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+appendRow.addEventListener('click', (e) => {
+  e.preventDefault();
+  e.stopPropagation();
+
+  const quantityOfRows = field.querySelectorAll('tr').length;
+
+  if (quantityOfRows < 10) {
+    const newRow = field.querySelector('tr').cloneNode(true);
```

Comment:
Ensure that there is at least one 'tr' element in the field before calling `cloneNode(true)`. If there are no rows initially, this will throw an error.